### PR TITLE
Migration to django-popolo part 1

### PR DESCRIPTION
**Pull request by mhl**
This is the first part of: https://github.com/ciudadanointeligente/write-it/pull/1181 for review. See that pull request for the context for this pull request.

It adds django-popolo models in parallel with the popit-django models that we're intending to replace.

### CODE REVIEW

File: instance/models.py [hunk 1]
```diff
@@ -147,6 +156,7 @@ class WriteitInstancePopitInstanceRecord(models.Model):
         )
     writeitinstance = models.ForeignKey(WriteItInstance)
     popitapiinstance = models.ForeignKey(ApiInstance)
+    popolo_source = models.ForeignKey(PopoloSource, null=True, blank=True)
```
Comment (chrismytton) [inline instance/models.py]: I'm getting an error on this line when I run the tests:

```
(writeit-virtualenv)vagrant@precise64:/vagrant$ ./manage.py test nuntium contactos mailit
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
    utility.execute()
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute
    django.setup()
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/__init__.py", line 21, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File "/home/vagrant/writeit-virtualenv/local/lib/python2.7/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/vagrant/instance/models.py", line 149, in <module>
    class WriteitInstancePopitInstanceRecord(models.Model):
  File "/vagrant/instance/models.py", line 159, in WriteitInstancePopitInstanceRecord
    popolo_source = models.ForeignKey(PopoloSource, null=True, blank=True)
NameError: name 'PopoloSource' is not defined
```

I'm guessing this line has snuck into this pull request unintentionally? :)

File: instance/models.py [hunk 2]
```diff
@@ -43,7 +47,7 @@ def relate_with_persons_from_popit_api_instance(self, popit_api_instance):
         except Exception, e:
             self.do_something_with_a_vanished_popit_api_instance(popit_api_instance)
             return (False, e)
-        persons = Person.objects.filter(api_instance=popit_api_instance)
+        persons = PopoloPerson.objects.filter(api_instance=popit_api_instance)
```
Comment (chrismytton) [inline instance/models.py]: It seems like this line should probably not be changed until _after_ the data has been migrated in the later commits?

File: nuntium/models.py [hunk 3]
```diff
@@ -168,7 +169,7 @@ def recently_confirmated(self):
 
     @property
     def people(self):
-        people = Person.objects.filter(
+        people = PopoloPerson.objects.filter(
```
Comment (chrismytton) [inline nuntium/models.py]: Same with this line, I think this should probably only be changed once the data has been migrated in later commits?

Comment (mhl) [inline nuntium/models.py]: Yes, right again.

File: instance/models.py (hunk 2 above)
Comment (mhl) [inline instance/models.py]: Yes, sorry, that should be in a later PR.

File: instance/models.py (hunk 1 above)
Comment (mhl) [inline instance/models.py]: In fact it's just missing the import - we need this foreign key to be there for the datamigration which will populate it with the equivalent PopoloSource to the old ApiInstance. I've pushed a fixup for this and the other accidental s/Person/PopoloPerson/ changes that you mentioned.

**Pull request merged**